content/applications/essentials/activities.rst

```diff
@@ -2,238 +2,348 @@
 Activities
 ==========
 
-*Activities* are follow-up tasks tied to a record in an Odoo database. Activities can be scheduled
-on any page of the database that contains a chatter thread, kanban view, list view, or activities
-view of an application.
+*Activities* are follow-up tasks tied to a record in an Odoo database.
+
+.. _activities/important:
```

Comment:
Suggestion to give this custom anchor a more descriptive name:
```suggestion
.. _activities/icon-display:
```
Or remove it since it is not used in this doc (however it is helpful to leave it for future references 🙂 )

Reply:
I was just checking that- i don't use this since I don't reference this anymore, so it's getting the ax!